src/longest_increasing_subsequence/dynamic_programming.jl

```diff
@@ -0,0 +1,58 @@
+"""
+    lis(arr::Array{Int}, ::Val{:dp})
```

Comment:
Can you tell me what the `::Val{:dp}` is for? I have not seen its uses in the tests nor it was mentioned what it is in the docstring

Reply:
nvm i understood just now.